src/main/java/kr/codesquad/Ladder.java

```diff
@@ -0,0 +1,26 @@
+package kr.codesquad;
+
+public class Ladder {
+    private final int participant;
+    private final int height;
+    private boolean[][] ladderMap;
+
+    public Ladder(int participant, int height) {
+        this.participant = participant;
+        this.height = height;
+        ladderMap = new boolean[height][participant - 1];
+        initLadderMap();
+    }
+
+    private void initLadderMap() {
```

Comment:
```suggestion
    private void initMap() {
```

클래스명과 메서드 이름에 겹치는 단어는 피하는 게 좋습니다.

Reply:
클래스 이름짓기에 더 유의하겠습니다. 감사합니다.